### PR TITLE
features/bash

### PR DESCRIPTION
**Pull request by davidhousesrodriguez**

### CODE REVIEW

**Comment by manuelbarzi** [comment]
REDO the branch. you have added extra files from arrays and strings that should not be here

File: staff/david-casas/bash/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+```
+## touch (creates an empty file)
+
+```sh
+$ touch hello.world
+```
+## mkdir(creates a directory / folder)
+
+```sh
+$ mkdir
```
Comment (iConceali) [inline staff/david-casas/bash/README.md]: You should put in the example the name of the folder you are going to create. Just like you did in #touch

File: staff/david-casas/bash/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+## cd (change directory)
+
+```sh
+$ cd workspace
+```
+
+## mv (move / rename file or folder)
+
+```sh
+$ mv
```
Comment (iConceali) [inline staff/david-casas/bash/README.md]: Here you have to put the complete command.
If it is to rename: $mv file1 file2
If it is to move: $mv file1 /folder1/file1

**Comment by davidhousesrodriguez** [comment]
> REDO the branch. you have added extra files from arrays and strings that should not be here

**Comment by davidhousesrodriguez** [comment]
> arrays

it´s supossed to be done

File: staff/david-casas/bash/README.md (hunk 1 above)
Comment (davidhousesrodriguez) [reply staff/david-casas/bash/README.md]: i think is already done

File: staff/david-casas/bash/README.md (hunk 2 above)
Comment (davidhousesrodriguez) [reply staff/david-casas/bash/README.md]: done as well

**Comment by joelsrzalvarez** [comment]
I see everything ok David ;)

**Pull request merged**